README.md

```diff
@@ -85,7 +85,7 @@ The returned value will typically be the `fromNode`. However, in situations wher
 
 Supported options (all optional):
 
-- **getNodeKey** (`Function(node)`) - Called to get the `Node`'s unique identifier. This is used by `morphdom` to rearrange elements rather than creating and destroying an element that already exists. This defaults to using the `Node`'s `id` attribute.
+- **getNodeKey** (`Function(node)`) - Called to get the `Node`'s unique identifier. This is used by `morphdom` to rearrange elements rather than creating and destroying an element that already exists. This defaults to using the `Node`'s `id` property. (Note that forms must not have fields with `name="id"`.)
```

Comment:
What do you think about this in between parentheses?  `Form controls shouldn't be given a name that are the same as other form properties.`

Reply:
I was wary of being unspecific, but that might have been overly cautious. I've tweaked the phrasing in fd939820be2cdd674b0bc8ce8c715f3ac1f9d5fa, but feel free to go with your version instead; either is fine with me.